helm/fiftyone-teams-app/README.md

```diff
@@ -94,11 +94,13 @@ documentation before completing your upgrade.
 Voxel51 recommends upgrading your deployment using
 [`legacy` authentication mode](https://docs.voxel51.com/teams/pluggable_auth.html#legacy-mode)
 and
-[migrating](https://docs.voxel51.com/teams/pluggable_auth.html#migrating-from-legacy-to-internal-mode)
+migrating
```

Comment:
Lines 96-98 should be changed to 


```
and migrating to
```

GitHub won't let me make a suggestion on a deleted line